doc/data/messages/i/invalid-character-carriage-return/bad.py

```diff
@@ -0,0 +1,7 @@
+# pylint: disable=trailing-whitespace, missing-module-docstring, trailing-newlines, missing-final-newline, syntax-error
```

Comment:
You don't need that, all message except the one we document and fatal one are disabled :)

Reply:
Changed it